.github/workflows/publish.yml

```
@@ -0,0 +1,28 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
```

I use `published` here.

Does `created` trigger already when creating a draft release?

If so, is it "safer" to use `published`?

```suggestion
    types: [published]
```